src/strings/sv.json

```diff
@@ -574,6 +574,7 @@
     "LabelOptionalNetworkPathHelp": "Om denna mappen delas på ditt nätverk, kan den delade sökvägen tillåta klienter på andra enheter att streama mediafiler direkt. Till exempel, {0} eller {1}.",
     "LabelOriginalAspectRatio": "Ursprungligt bildförhållande",
     "LabelOriginalTitle": "Originaltitel",
+    "LabelOriginalLanguage": "Originalspråk",
```

Comment:
Undo the changes in the `sv.json` file. They must be made via Weblate. Otherwise it may cause merge conflicts.